pkg/client/cache/reflector.go

```diff
@@ -265,6 +265,7 @@ func (r *Reflector) ListAndWatch(stopCh <-chan struct{}) error {
 			switch err {
 			case io.EOF:
 				// watch closed normally
+				glog.V(1).Infof("%s: Watch for %v closed normally (with EOF)", r.name, r.expectedType)
```

Comment:
It happens whenever the server closes the watch (random timeout > 5m i think), we want to do a full relist periodically and we can't guarantee that the watch will be open always anyway.